### ISSUE TITLE
Record too big error does not raise an exception

### ISSUE THREAD

**Issue by cdussud** [algolia/algoliasearch-rails#207]
When my record is too large to index I see a message in the console
```
{"message":"Record at the position 0 is too big size=119344 bytes. Contact us if you need an extended quota","position":0,"status":400}
```
Unfortunately no exception is thrown. Since the item can't be indexed we'd like for our error monitoring tools to pick this up.

**Comment by redox** [comment]
> Unfortunately no exception is thrown. Since the item can't be indexed we'd like for our error monitoring tools to pick this up.

Hmmm, oh it should throw an exception. Did you configure the `algoliasearch` block with a `raise_on_failure` option?

**Comment by redox** [comment]
Arg, I reproduce the issue in a test; investigating 👍

**Comment by cdussud** [comment]
Any updates on this?

**Comment by redox** [comment]
> Any updates on this?

Had issues with Travis, trying it again now.

**Comment by redox** [comment]
OK, I got confused by Rails 3.2 -> I forgot this versions of Rails was swallowing the exceptions raised in an `after_commit` callback... Will merge + release as soon as Travis is green @cdussud  👍

**Comment by redox** [comment]
This has been released in `1.19.0`. Thank you for your patience @cdussud 👍

**Comment by cdussud** [comment]
Thanks @redox!
